app/models/track.rb

```diff
@@ -41,7 +41,7 @@ def git
     Git::Track.new(synced_to_git_sha, repo_url: repo_url)
   end
 
-  # TODO: Drive from config.json
+  # TODO: Erik: Drive from config.json
```

Comment:
I assume that we'll use the `status.concept_exercises` key from the `config.json` for this? Or is there more to it?

Reply:
Yeah, basically. Although I think we might want to rename the key.

Reply:
> Although I think we might want to rename the key.

That will go over well, another bulk PR :P